.github/workflows/macos-ci.yml

```diff
@@ -55,18 +60,16 @@ jobs:
         if: ${{ matrix.homebrew-gl }}
         run: |
           brew install mesa mesa-glu freeglut
-          ln -s /usr/local/include/GL /usr/local/include/OpenGL
-          ln -s /usr/local/include/GL /usr/local/include/GLUT
-        # ln -s /usr/local/lib/libGL.dylib /usr/local/lib/libOpenGL.dylib
-        # find /usr/local/lib/ -iname '*gl*.dylib'
+          ln -s "$(brew --prefix)/include/GL" "$(brew --prefix)/include/OpenGL"
+          ln -s "$(brew --prefix)/include/GL" "$(brew --prefix)/include/GLUT"
```

Comment:
Note that Homebrew installs stuff in a different default parent directory on Apple Silicon vs. x86_64. And the Homebrew directory can be changed from the default, also. Using `$(brew --prefix)` should take care of all of those cases.